src/main/java/org/opensrp/repository/postgres/SettingRepositoryImpl.java

```diff
@@ -506,11 +519,16 @@ public void add(SettingConfiguration entity) {
 			pgSettings = convert(entity, id);
 		}
 
-		checkWhetherMetadataExistsBeforeSave(entity, settings, pgSettings);
+		try {
+			checkWhetherMetadataExistsBeforeSave(entity, settings, pgSettings);
+		}
+		catch (SQLException e) {
+			e.printStackTrace();
+		}
```

Comment:
Remove this try catch since error is not handled

Reply:
Changing this to a RuntimeException